packages/storybook/stories/components/badge/badge.stories.ts

```diff
@@ -114,6 +132,20 @@ export const Color: StoryObj = {
   `,
 };
 
+export const WithLeftIcon: StoryObj = {
+  tags: ['isHidden'],
+  render: () => html`
+    <ods-badge label="Home left" icon="${ODS_ICON_NAME.home}" icon-alignment="${ODS_BADGE_ICON_ALIGNMENT.left}">Badge left</ods-badge>
+  `,
+};
+
+export const WithRightIcon: StoryObj = {
```

Comment:
those two should be added to the Technical page under Examples (or perhaps under one story Icon Alignement instead with both in it?)